crates/re_log_types/src/lib.rs

```diff
@@ -88,39 +90,32 @@ macro_rules! impl_into_enum {
 // ----------------------------------------------------------------------------
 
 /// A unique id per recording (a stream of [`LogMsg`]es).
-#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Clone, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
-pub struct RecordingId(uuid::Uuid);
-
-impl nohash_hasher::IsEnabled for RecordingId {}
-
-// required for [`nohash_hasher`].
-#[allow(clippy::derived_hash_with_manual_eq)]
-impl std::hash::Hash for RecordingId {
-    #[inline]
-    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
-        state.write_u64(self.0.as_u128() as u64);
-    }
-}
+pub struct RecordingId(Arc<String>);
 
 impl Default for RecordingId {
     fn default() -> Self {
-        Self::ZERO
+        Self::unknown()
     }
 }
 
 impl RecordingId {
-    /// The recording id:s given to recordings that don't have an ID.
-    pub const ZERO: RecordingId = RecordingId(uuid::Uuid::nil());
+    const UNKNOWN: &str = "UNKNOWN";
+
+    #[inline]
+    pub fn unknown() -> Self {
+        RecordingId::UNKNOWN.into()
+    }
```

Comment:
the indirection doesn't really add anything, does it?

```suggestion
    #[inline]
    pub fn unknown() -> Self {
        "UNKNOWN".into()
    }
```